Crypto/SecureDigest.cc

```diff
@@ -82,4 +87,49 @@ namespace litecore {
 #endif
     }
 
+#undef _CONTEXT
+#ifdef USE_COMMON_CRYPTO
+#include <CommonCrypto/CommonDigest.h>
+#define _CONTEXT ((CC_SHA256_CTX*)_context)
```

Comment:
Is this macro necessary? All its uses below are already inside `#ifdef USE_COMMON_CRYPTO`, so the abstraction isn't needed.

Reply:
`_CONTEXT` is defined differently for CommonCrypto vs mbed.  I just reused what was happening with SHA1